File: go/samples/people_picker_sample.go
package main

import "github.com/rido-min/FluentCards/go/fluentcards"

func createPeoplePickerCard() fluentcards.Card {
	return fluentcards.NewAdaptiveCardBuilder().
		WithVersion("1.6").
		AddInputChoiceSet(func(i *fluentcards.InputChoiceSetBuilder) {
			i.WithID("people-picker").
				WithLabel("Select users in the whole organization").
				WithIsMultiSelect(true).
				WithValue("user1,user2").
				WithChoicesData("graph.microsoft.com/users")
		}).
		AddAction(func(a *fluentcards.ActionBuilder) {
			a.Submit("Submit")
		}).
		Build()
}

File: dotnet/samples/FluentCards.Samples/PeoplePickerSample.cs
namespace FluentCards.Samples;

/// <summary>
/// Demonstrates creating a people picker card using dynamic data queries.
/// </summary>
public static class PeoplePickerSample
{
    /// <summary>
    /// Creates a people picker card that searches users from Microsoft Graph.
    /// </summary>
    /// <returns>An Adaptive Card with a dynamic people picker and submit action.</returns>
    public static AdaptiveCard CreatePeoplePickerCard()
    {
        return AdaptiveCardBuilder.Create()
            .WithVersion("1.6")
            .AddInputChoiceSet(i => i
                .WithId("people-picker")
                .WithLabel("Select users in the whole organization")
                .IsMultiSelect()
                .WithValue("user1,user2")
                .WithChoicesData("graph.microsoft.com/users"))
            .AddAction(a => a
                .Submit("Submit"))
            .Build();
    }
}
